### DISCUSSION TITLE
How many quotes are in the database?

### DISCUSSION THREAD

**Discussion by adambirds** [lukePeavey/quotable#115] [Questions]
I was asked by a user how many quotes are in the database all together as he was just interested in how many different quotes we could potentially get.

**Comment by lukePeavey** [comment]
**Quotes: 1885
Authors: 700**

To check the total number of quotes at any given time, you can make a request to the `/quotes` endpoint.  The `totalCount` property in the response tells you the total number of quotes in the database. 

```HPPT
GET https://api.quotable.io/quotes
```

```json
{
  "count": 1,
  "totalCount": 1885,
  "page": 1,
  "totalPages": 1885,
  "lastItemIndex": 1,
  "results": []
}
```
